Apsitvarkom.UnitTests/Models/Validation/PollutedLocationValidationTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Apsitvarkom.Models;
+using FluentValidation;
+
+namespace Apsitvarkom.UnitTests.Models.Validation
+{
+    internal class PollutedLocationValidationTests
+    {
+        private static readonly IValidator<PollutedLocation> Validator = new PollutedLocationValidator();
+        private static readonly Guid dummyGuid = Guid.NewGuid();
```

Comment:
```suggestion
        private static readonly Guid DummyGuid = Guid.NewGuid();
```

[This](https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/d53b55ey(v=vs.71)?redirectedfrom=MSDN) old MSDN article suggests using PascalCase for static variables.